config/vendor-api-v1.yml

```diff
@@ -594,6 +594,16 @@ components:
           maxLength: 256
           description: The candidate’s UK residency status indicating their right to work and study in the UK. Possible values include "UK Citizen", "Irish Citizen", "Candidate needs to apply for permission to work and study in the UK" and "Candidate does not know". The candidate can also provide details as free text for example "Settled status".
           example: UK Citizen
+        uk_residency_status_code:
+          type: string
+          maxLength: 1
+          description: Single alphabetical character code for the candidate’s UK residency status indicating their right to work and study in the UK.
```

Comment:
Should we add the following code mapping in the description, as I reckon they'd like to see the which code maps to which status 🤔  ? Not sure if there is a better place to do this

```
A- UK Citizen
B- Irish Citizen 
C- Candidate needs to apply for permission to work and study in the UK
D- Candidate's free text response 

```